velox/functions/prestosql/tests/BinaryFunctionsTest.cpp

```diff
@@ -583,4 +583,44 @@ TEST_F(BinaryFunctionsTest, toBigEndian64) {
       toBigEndian64((int64_t)-9223372036854775807 - 1));
 }
 
+TEST_F(BinaryFunctionsTest, toIEEE754Bits64) {
+  const auto toIEEE754Bits64 = [&](std::optional<double> value) {
+    return evaluateOnce<std::string, double>("to_ieee754_64(c0)", value);
+  };
+
+  EXPECT_EQ(std::nullopt, toIEEE754Bits64(std::nullopt));
```

Comment:
Please add tests for std::numeric_limits<T>::max(), std::numeric_limits<T>::min(), std::numeric_limits<T>::infinity(), std::numeric_limits<T>::quiet_Nan(),

Reply:
done